nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>Read this tutorial that Jeremy authored yesterday https://nbdev.fast.ai/explanations/showdoc.html and perhaps link to it instead or update your explanation based on that.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>